### DISCUSSION TITLE
Enhancement: Resolutions other thean 720p

### DISCUSSION THREAD

**Discussion by libauror-a** [FunkinCrew/Funkin#1677] [Ideas]
The game currently runs at 720p and no other resolution, because of that fullscreen looks blurry because it's just resizing a 720p game to your screens resolution (like 1080p). Add multiple resolutions (or make the game adapt to your resolution) so it can fit your monitors resolution and not look blurry.

**Comment by ACrazyTown** [comment]
I think making the game adapt to the resolution is the best option tbh. Pretty sure just changing the values in Main.hx and Project.xml would require every Object that uses X and Y params to be adjusted for said resolution, they're probably gonna add it in the future but it might be a bit tricky considering every asset was made in 720p (i think, don't quote me on that)

**Comment by havi11368** [comment]
Bad idea because all it does is reducing your FPS to 30.

**Comment by ghost** [comment]
> would require every Object that uses X and Y params to be adjusted for said resolution

The notes, a bit, but there is a scale property in main.hx that does most of the job

**Comment by ghost** [comment]
> Bad idea because all it does is reducing your FPS to 30.

Trust me, after changing the values in Project.xml and main.hx, the FPS was the same.

**Comment by libauror-a** [comment]
> > Bad idea because all it does is reducing your FPS to 30.
> 
> Trust me, after changing the values in Project.xml and main.hx, the FPS was the same.

yes because you need to change the stage fps

**Comment by ninjamuffin99** [comment]
is somethin we'd prob work on over the years. It's not just a matter of changing a few values, it's ripping out and replacing all the art assets with higher res art, and still being performant.

will take a while but its in my mind since the game was first coded
